### DISCUSSION TITLE
Whisper problem in react native

### DISCUSSION THREAD

**Discussion by sirramin** [microsoft/onnxruntime#18447] [Show & Tell]
I implemented whisper but the output data is empty when using android recorder data:
[https://github.com/sirramin/onnxruntime-whisper-react-native](https://github.com/sirramin/onnxruntime-whisper-react-native)

I can get output using data from sample web implementation but when using react native recording plugins there is no output.

**Comment by danieloi** [comment]
did you ever figure this out? Were you using a wav file?

**Reply by sirramin** [reply]
I remember it was working in the last version of my code
